public/app/features/dashboard/dashgrid/DashboardGrid.tsx

```diff
@@ -227,59 +233,24 @@ export class DashboardGrid extends PureComponent<Props, State> {
               return null;
             }
 
+            // Disable draggable if mobile device, solving an issue with unintentionally
+            // moving panels. https://github.com/grafana/grafana/issues/18497
+            // theme.breakpoints.md = 769
             const draggable = width <= 769 ? false : isEditable;
```

Comment:
Cannot we access the theme value here using the HoC `withTheme2`? Just consider the case we modify the breakpoint.

Reply:
@ivanortegaalba no need, can just access theme directly here via config